react/features/analytics/handlers/AmplitudeHandler.js

```diff
@@ -0,0 +1,71 @@
+import amplitude from 'amplitude-js';
+
+import { getJitsiMeetGlobalNS } from '../../base/util';
+
+import AbstractHandler from './AbstractHandler';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Analytics handler for Amplitude.
+ */
+class AmplitudeHandler extends AbstractHandler {
+    /**
+     * Creates new instance of the Amplitude analytics handler.
+     *
+     * @param {Object} options -
+     * @param {string} options.amplitudeAPPKey - The Amplitude app key required
+     * by the Amplitude API.
+     */
+    constructor(options) {
+        super();
+
+        const { amplitudeAPPKey } = options;
+
+        if (!amplitudeAPPKey) {
+            logger.warn(
+                'Failed to initialize Amplitude handler, no tracking ID');
+
+            return;
+        }
+
+        this._enabled = true;
+
+        amplitude.getInstance().init(amplitudeAPPKey);
```

Comment:
why not cache this instance in order to avoid calling `getInstance()` every time?

Reply:
This was my initial implementation and it turns out there is a problem somewhere in the Amplitude SDK. Basically what happens if we cache it, is that no requests are sent by the Amplitude SDK.